modules/mediawiki/manifests/nginx.pp

```diff
@@ -1,6 +1,7 @@
-# MediaWiki nginx config using hiera
+# === Class mediawiki::nginx
```

Comment:
I assume this all requires nginx + php installed

Reply:
None of the other modules explicitly requires so I don't think we should here.